src/pages/Chat/ChattingPage.tsx

```diff
@@ -0,0 +1,131 @@
+import { useEffect, useState, useRef } from "react"
+import styled from "styled-components"
+import TopBarIcons from "../../components/TopBarIcons"
+import ChatInput from "./ChatInput"
+import friendProfile from "../../assets/img/friendProfile.png"
+import myProfile from "../../assets/img/myProfile.png"
+import dayjs from "dayjs"
+import ChatBubbleReceiver from "../Chat/ChatBubbleReceiver"
+import ChatBubbleSender from "./ChatBubbleSender"
+import { useRecoilState } from 'recoil';
+import { chatDataState } from '../../atoms';
+import { useParams } from 'react-router-dom';
+import dummyData from "../../data/dummyData.json"
+
+interface ChatType {
+  //채팅 방 데이터
+  r_id: number; //채팅방(room) 고유 id
+  isGroup: boolean; //grop일 경우 group icon을 띄우기 위함
+  r_name: string; //채팅방 이름
+  r_profile?: string; //수신자 프로필 url
+  //채팅 1개 당 데이터
+  chat : {
+    c_id: number | null; //채팅 1개당 id
+    sender: string; //발신자
+    receiver: string; //수신자
+    value: string; //텍스트 입력값
+    time: string //텍스트 보낸 시간
+  }[]
+}
+
+
+function ChattingPage() {
+  const { id } = useParams<{ id: string }>();
+  const chatRoomJson = dummyData.find(chat => chat.r_id === Number(id));
+  console.log(chatRoomJson);
+
+  const [value,  setValue] = useState<string>(''); //텍스트 입력값을 넣어주기 위한 state
+
+
+  {/*const [chatData, setChatData] = useRecoilState<ChatType>(chatDataState);
+
+  useEffect(() => {
+    if (chatRoomJson) {
+      setChatData(chatRoomJson);
+    }
+  }, []);
+  
+  console.log(chatData);
+
+  
+  //스크롤 구현
+  const chatEndRef = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    chatEndRef.current?.scrollIntoView({ behavior: "smooth" });
+  }, [chatData]);
+
+
+*/}
```

Comment:
코드에 여러 주석 처리된 섹션이 있으며, 이들이 현재 사용되지 않거나 향후 사용을 위해 남겨두었다면, 각 부분에 대한 명확한 주석을 추가하거나 필요 없다면 제거하는 것이 좋습니다.!

Reply:
채팅 데이터 변경시 스크롤이 구현되는 것은 이런느낌으로 수정하면 될 것 같아요

```suggestion
useEffect(() => {
  const scrollToBottom = () => {
    chatEndRef.current?.scrollIntoView({ behavior: "smooth" });
  };
  scrollToBottom();
}, [chatRoomJson?.chat]); // 의존성 배열을 chatRoomJson.chat으로 변경


<div ref={chatEndRef} style={{ width: "100%", height: "1px" }}/>
```